doc/source/development/rolling_operations_with_numba.rst

```diff
@@ -0,0 +1,234 @@
+Rolling Operations with Numba
+=============================
+
+Summary
+-------
+
+This proof of concept (POC) demonstrates using `Numba <http://numba.pydata.org/>`_ instead of ``Cython``
+to compute ``rolling.mean`` and ``rolling.apply`` without introducing any user facing API changes.
```

Comment:
use the full refernce here (i left a comment about this below, but here is enough)